install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -611,3 +611,11 @@ data:
 {{- else if ( ne $crdWaitTimeout "5m" ) }}
   crd-wait-timeout: {{ $crdWaitTimeout | quote }}
 {{- end }}
+
+{{- if hasKey .Values "enableK8sEndpointSlice" }}
+  enable-k8s-endpoint-slice: {{ .Values.enableK8sEndpointSlice | quote }}
+{{- end }}  
+{{- if not .Values.enableK8sEndpointSlice }}
```

Comment:
Why do we need this case? If users want to disable it, then they should `--set config.enableK8sEndpointSlice=false`?

Reply:
I'd keep it simple - in order to disable the feature users would be required to explicitly set the value.

Reply:
```
{{- if hasKey .Values "enableK8sEndpointSlice" }}
  enable-k8s-endpoint-slice: {{ .Values.enableK8sEndpointSlice | quote }}
{{- end }}  
```

Reply:
Just did the force push